packages/frontend/src/redux/actions/staking.js

```diff
@@ -553,8 +551,7 @@ export const handleUpdateCurrent = (accountId) => async (dispatch, getState) =>
     dispatch(staking.updateCurrent({ currentAccount }));
 };
 
-export const getValidatorFarmData = (validator, accountId) => async (dispatch, getState) => {
-
+export const getValidatorFarmData = createAsyncThunk('staking/setValidatorFarmData', async ({ validator, accountId }, { dispatch }) => {
```

Comment:
```suggestion
export const getValidatorFarmData = createAsyncThunk('staking/getValidatorFarmData', async ({ validator, accountId }, { dispatch }) => {
```
typo?

Reply:
Either works but was intended to replace the removed action of the same name 😄 . I guess since there's a fetch this time `getValidatorFarmData` makes more sense 👍